.gitignore

```diff
@@ -11,3 +11,4 @@ __pycache__/
 *.html
 /psycopg_binary/
 .vscode
+.venv
```

Comment:
This file doesn't end in a newline. But no problem: I will make the fix in master and rebase this on, fixing the conflict.

Reply:
cool